client/src/App.css

```diff
@@ -44,8 +44,8 @@
 }
 
 .fluid.container {
-  margin-left: 10px!important;
-  margin-right: 10px!important;
+  margin-left: 10px !important;
```

Comment:
Do you really need the !important here? Usually it's a sign that a more idiomatic styling is possible.